generators/client/templates/angular/package.json.ejs

```diff
@@ -44,7 +44,7 @@ limitations under the License.
     <%_ if (clientTheme !== 'none') { _%>
     "bootswatch": "VERSION_MANAGED_BY_CLIENT_ANGULAR#dependencies#bootstrap",
     <%_ } _%>
-    "moment": "2.27.0",
+    "dayjs": "1.9.3",
```

Comment:
```suggestion
    "dayjs": "VERSION_MANAGED_BY_CLIENT_ANGULAR",
```

And add dayjs to package.json in the same folder.
Should be done at all frameworks

Reply:
Ok, I understand how it works now, so I pushed a new version, but I used the `VERSION_MANAGED_BY_CLIENT_COMMON` and I put the version in the common package.json so that all versions are managed at the same time

Reply:
Great.